Error: 'va_copy' / 'va_end' not recognised as identifier.

**Environment (please complete the following information):**
 - OS/OS Version: Windows 10
 - Source Version: master/v2.4.0
 - Unity Version: 2019.4.27.f1

**Describe the bug**
When building a windows executable of the full of all scenes or just the `SSD` scene, using `IL2CPP`, an error is thrown, mentioning:
```bash
il2cpp.exe didn't catch exception: Unity.IL2CPP.Building.BuilderFailedException: tflite_unity_helper.cpp
D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp(10): error C3861: 'va_copy': identifier not found
D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp(12): error C3861: 'va_end': identifier not found
```
Full error included in the following comment.

When I examine the `tflite_unity_helper.cpp` file, it appears to be that it does not contain `#include <cstdarg>`, which is where `va_copy` and `va_end` are defined. Is there a way to add it, and could that be the issue?

I'll include the full file in a comment below as well.

**To Reproduce**
Steps to reproduce the behavior:
1. Press Build Settings
2. Select `PC, Mac & Linux Standalone`
3. Under `Player Settings` / `Other`, add a tick near `IL2CPP` (I'm new to this - assuming this is the right way to build it.)
4. The building fails with the error.

**Expected behavior**
The building succeeds / the `va_copy` and `va_end` are found.

**Extra**
Thanks a lot for making this repo. For someone who's new to both Unity and TFLite, it's amazing to find clear examples such as this :)

Here's the error in full:

```bash
Exception: C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp/build/deploy/net471/il2cpp.exe did not run properly!

Failed running "C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp/build/deploy/net471/il2cpp.exe" --convert-to-cpp --emit-null-checks --enable-array-bounds-check --dotnetprofile="unityaot" --compile-cpp --libil2cpp-static --platform="WindowsDesktop" --architecture="x64" --configuration="Release" --outputpath="D:\Unity Projects\tf-lite-unity-sample\Temp/StagingArea/Data\Native\GameAssembly.dll" --cachedirectory="D:\Unity Projects\tf-lite-unity-sample\Assets\..\Library/il2cpp_cache" --profiler-report --map-file-parser="C:/Program Files/Unity/Hub/Editor/2019.4.27f1/Editor/Data/Tools/MapFileParser/MapFileParser.exe" --directory="D:/Unity Projects/tf-lite-unity-sample/Temp/StagingArea/Data/Managed" --generatedcppdir="D:/Unity Projects/tf-lite-unity-sample/Temp/StagingArea/Data/il2cppOutput"  

stdout:
Building GameAssembly.dll with MsvcDesktopToolChain
	Msvc Install Version: 15.0
	Msvc Install SDK Directory: C:\Program Files (x86)\Windows Kits\10
	Msvc Linker Path: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX64\x64\link.exe
	Msvc Compiler Path: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX64\x64\cl.exe

	Output directory: D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\Native
	Cache directory: D:\Unity Projects\tf-lite-unity-sample\Library\il2cpp_cache
il2cpp.exe didn't catch exception: Unity.IL2CPP.Building.BuilderFailedException: tflite_unity_helper.cpp
D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp(10): error C3861: 'va_copy': identifier not found
D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp(12): error C3861: 'va_end': identifier not found

Invocation was: Executable: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX64\x64\cl.exe"
Arguments: "D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp" /nologo /c /bigobj /W3 /Zi /EHs /GR- /Gy /utf-8 /wd4102 /wd4800 /wd4056 /wd4190 /wd4723 /wd4467 /wd4503 /wd4996 /wd4200 /Ox /Oi /Oy- /GS- /Gw /GF /Zo /MT /DNET_4_0 /DUNITY_AOT /DIL2CPP_MONO_DEBUGGER_DISABLED /DGC_NOT_DLL /DRUNTIME_IL2CPP /D_WIN32 /DWIN32 /DWIN32_THREADS /D_WINDOWS /DWINDOWS /D_UNICODE /DUNICODE /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /D_WINSOCK_DEPRECATED_NO_WARNINGS /DNOMINMAX /D_NDEBUG /DNDEBUG /DWINDOWS_SDK_BUILD_VERSION=19041 /DWINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\libil2cpp" /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\libil2cpp" /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\external\bdwgc\include" /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\external\xxHash" /I"D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput" /I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\shared" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\winrt" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\ucrt" /Fo"D:\Unity Projects\tf-lite-unity-sample\Library\il2cpp_cache\4DF896D3F41D7212E2EB19362D908399.obj" /Fd"D:\Unity Projects\tf-lite-unity-sample\Library\il2cpp_cache\4DF896D3F41D7212E2EB19362D908399.pdb"
EnvArg key: PATH value: C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x86;C:\Program Files (x86)\Windows Kits\10\bin\x86;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX64\x64

   at Unity.IL2CPP.Building.CppProgramBuilder.BuildAllCppFiles(IEnumerable`1 sourceFilesToCompile, IBuildStatisticsCollector statisticsCollector)
   at Unity.IL2CPP.Building.CppProgramBuilder.Build(IBuildStatistics& statistics)
   at il2cpp.Program.DoRun(String[] args, List`1 foundAssemblies)
   at il2cpp.Program.Run(String[] args, Boolean setInvariantCulture)
   at il2cpp.Program.Main(String[] args)
stderr:

Unhandled Exception: Unity.IL2CPP.Building.BuilderFailedException: tflite_unity_helper.cpp
D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp(10): error C3861: 'va_copy': identifier not found
D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp(12): error C3861: 'va_end': identifier not found

Invocation was: Executable: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX64\x64\cl.exe"
Arguments: "D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput\tflite_unity_helper.cpp" /nologo /c /bigobj /W3 /Zi /EHs /GR- /Gy /utf-8 /wd4102 /wd4800 /wd4056 /wd4190 /wd4723 /wd4467 /wd4503 /wd4996 /wd4200 /Ox /Oi /Oy- /GS- /Gw /GF /Zo /MT /DNET_4_0 /DUNITY_AOT /DIL2CPP_MONO_DEBUGGER_DISABLED /DGC_NOT_DLL /DRUNTIME_IL2CPP /D_WIN32 /DWIN32 /DWIN32_THREADS /D_WINDOWS /DWINDOWS /D_UNICODE /DUNICODE /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /D_WINSOCK_DEPRECATED_NO_WARNINGS /DNOMINMAX /D_NDEBUG /DNDEBUG /DWINDOWS_SDK_BUILD_VERSION=19041 /DWINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\libil2cpp" /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\libil2cpp" /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\external\bdwgc\include" /I"C:\Program Files\Unity\Hub\Editor\2019.4.27f1\Editor\Data\il2cpp\external\xxHash" /I"D:\Unity Projects\tf-lite-unity-sample\Temp\StagingArea\Data\il2cppOutput" /I"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\shared" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\winrt" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\ucrt" /Fo"D:\Unity Projects\tf-lite-unity-sample\Library\il2cpp_cache\4DF896D3F41D7212E2EB19362D908399.obj" /Fd"D:\Unity Projects\tf-lite-unity-sample\Library\il2cpp_cache\4DF896D3F41D7212E2EB19362D908399.pdb"
EnvArg key: PATH value: C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x86;C:\Program Files (x86)\Windows Kits\10\bin\x86;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX64\x64

   at Unity.IL2CPP.Building.CppProgramBuilder.BuildAllCppFiles(IEnumerable`1 sourceFilesToCompile, IBuildStatisticsCollector statisticsCollector)
   at Unity.IL2CPP.Building.CppProgramBuilder.Build(IBuildStatistics& statistics)
   at il2cpp.Program.DoRun(String[] args, List`1 foundAssemblies)
   at il2cpp.Program.Run(String[] args, Boolean setInvariantCulture)
   at il2cpp.Program.Main(String[] args)

UnityEditorInternal.Runner.RunProgram (UnityEditor.Utils.Program p, System.String exe, System.String args, System.String workingDirectory, UnityEditor.Scripting.Compilers.CompilerOutputParserBase parser) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditorInternal.Runner.RunManagedProgram (System.String exe, System.String args, System.String workingDirectory, UnityEditor.Scripting.Compilers.CompilerOutputParserBase parser, System.Action`1[T] setupStartInfo) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditorInternal.IL2CPPBuilder.RunIl2CppWithArguments (System.Collections.Generic.List`1[T] arguments, System.Action`1[T] setupStartInfo, System.String workingDirectory) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditorInternal.IL2CPPBuilder.ConvertPlayerDlltoCpp (UnityEditor.Il2Cpp.Il2CppBuildPipelineData data, System.String outputDirectory, System.String workingDirectory, System.Boolean platformSupportsManagedDebugging) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditorInternal.IL2CPPBuilder.Run () (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditorInternal.IL2CPPUtils.RunIl2Cpp (System.String stagingAreaData, UnityEditorInternal.IIl2CppPlatformProvider platformProvider, System.Action`1[T] modifyOutputBeforeCompile, UnityEditor.RuntimeClassRegistry runtimeClassRegistry) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
DesktopStandalonePostProcessor.SetupStagingArea (UnityEditor.Modules.BuildPostProcessArgs args, System.Collections.Generic.HashSet`1[T] filesToNotOverwrite) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
DesktopStandalonePostProcessor.PostProcess (UnityEditor.Modules.BuildPostProcessArgs args) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
Rethrow as BuildFailedException: Exception of type 'UnityEditor.Build.BuildFailedException' was thrown.
DesktopStandalonePostProcessor.PostProcess (UnityEditor.Modules.BuildPostProcessArgs args) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditor.Modules.DefaultBuildPostprocessor.PostProcess (UnityEditor.Modules.BuildPostProcessArgs args, UnityEditor.BuildProperties& outProperties) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditor.PostprocessBuildPlayer.Postprocess (UnityEditor.BuildTargetGroup targetGroup, UnityEditor.BuildTarget target, System.String installPath, System.String companyName, System.String productName, System.Int32 width, System.Int32 height, UnityEditor.BuildOptions options, UnityEditor.RuntimeClassRegistry usedClassRegistry, UnityEditor.Build.Reporting.BuildReport report) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

```

--- a/Packages/com.github.asus4.tflite/Plugins/tflite_unity_helper.cpp
+++ b/Packages/com.github.asus4.tflite/Plugins/tflite_unity_helper.cpp
@@ -1,5 +1,6 @@
 
 #include <iostream>
+#include <cstdarg>
 #include <cstring>
 
 extern "C" {

